codebuild/spec/buildspec_ubuntu.yml

```diff
@@ -25,6 +25,8 @@ phases:
     commands:
       - echo Entered the install phase...
       - echo "We need a test PPA for gcc-9."
+      # Update Yarn Debian key, see https://github.com/yarnpkg/yarn/issues/7866
+      - curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg  | apt-key add -
```

Comment:
This might be better used in https://github.com/awslabs/s2n/blob/main/codebuild/bin/install_ubuntu_dependencies.sh